ash/src/vk/extensions.rs

```diff
@@ -23945,25 +23990,32 @@ impl StructureType {
     pub const RENDER_PASS_CREATION_FEEDBACK_CREATE_INFO_EXT: Self = Self(1_000_458_002);
     pub const RENDER_PASS_SUBPASS_FEEDBACK_CREATE_INFO_EXT: Self = Self(1_000_458_003);
 }
-impl ExtExtension460Fn {
+impl LunargDirectDriverLoadingFn {
     #[inline]
     pub const fn name() -> &'static ::std::ffi::CStr {
-        unsafe { ::std::ffi::CStr::from_bytes_with_nul_unchecked(b"VK_EXT_extension_460\0") }
+        unsafe {
+            ::std::ffi::CStr::from_bytes_with_nul_unchecked(b"VK_LUNARG_direct_driver_loading\0")
+        }
     }
-    pub const SPEC_VERSION: u32 = 0u32;
+    pub const SPEC_VERSION: u32 = 1u32;
 }
 #[derive(Clone)]
-pub struct ExtExtension460Fn {}
-unsafe impl Send for ExtExtension460Fn {}
-unsafe impl Sync for ExtExtension460Fn {}
-impl ExtExtension460Fn {
```

Comment:
Would be nice if we could stop generating these non-bindings in the first place.

Reply:
#448, iirc.